sway_libs/src/signed_integers/i128.sw

```diff
@@ -0,0 +1,175 @@
+library i128;
+
+use std::u128::U128;
+
+/// The 128-bit signed integer type.
+/// Represented as an underlying U128 value.
+/// Actual value is underlying value minus 2 ^ 127
+/// Max value is 2 ^ 127 - 1, min value is - 2 ^ 127
+pub struct I128 {
+    underlying: U128,
+}
+
+pub enum Error {
```

Comment:
Have you considered moving this to its own errors.sw file? Would make sense if it's repeated